public/app/plugins/panel/graph/thresholds_form.html

```diff
@@ -38,8 +38,7 @@
             ng-options="f for f in ['custom', 'critical', 'warning', 'ok']"
             ng-change="ctrl.onThresholdTypeChange($index)"
             ng-disabled="ctrl.disabled"
-          >
```

Comment:
just making a note that the git diff is here because of some kind of linting or something that seems to have touched a lot of unrelated files, let's undo those changes before merging just to make it easier to see what we're touching/not touching

Reply:
This is done :)